waltz-web/src/main/java/com/khartec/waltz/web/endpoints/api/EntityEventEndpoint.java

```diff
@@ -0,0 +1,39 @@
+package com.khartec.waltz.web.endpoints.api;
+
+import com.khartec.waltz.model.entity_event.EntityEvent;
+import com.khartec.waltz.service.entity_event.EntityEventService;
+import com.khartec.waltz.web.ListRoute;
+import com.khartec.waltz.web.endpoints.Endpoint;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import static com.khartec.waltz.web.WebUtilities.getEntityReference;
+import static com.khartec.waltz.web.WebUtilities.mkPath;
+import static com.khartec.waltz.web.endpoints.EndpointUtilities.getForList;
+
+
+@Service
+public class EntityEventEndpoint implements Endpoint {
+
+    private static final String BASE_URL = mkPath("api", "entity", "event");
```

Comment:
v. minor - but other entity* endpoints are usually registered like:

- `entity-tag`
- `entity-statistic`
- `entity-search` 

(i.e. hyphenated, not `/` separated)

Reply:
Fixed